platform/windows/display_server_windows.cpp

```diff
@@ -83,11 +83,6 @@
 
 #define WM_INDICATOR_CALLBACK_MESSAGE (WM_USER + 1)
 
-#if defined(__GNUC__)
-// Workaround GCC warning from -Wcast-function-type.
-#define GetProcAddress (void *)GetProcAddress
-#endif
```

Comment:
I wonder if it wouldn't be better to put this in `typedefs.h` so it's always available.
Since it compiles fine on MSVC without `(void *)` it might be easy to miss in future additions.

Reply:
GCC build in CI should always catch feature additions (and I guess clang-cl as well from now), and I do not think all Windows specific files include `typedefs.h`, so not sure if it's better. Also, some parts of the code were alredy using direct `(void *)` cast.